docs/server/commands.rst

```diff
@@ -462,11 +462,12 @@ no actual data will be loaded:
 
 
 This command is capable of importing translations from several files in
-different formats at once:
+different formats at once, either specified directly or by passing its
+container directory:
```

Comment:
This command is capable of importing translations in multiple formats from several files and directories at once.